client/my-sites/checkout/checkout/credit-card-payment-box.jsx

```diff
@@ -154,7 +154,7 @@ class CreditCardPaymentBox extends React.Component {
 				<div className="checkout__secure-payment">
 					<div className="checkout__secure-payment-content">
 						<Gridicon icon="lock" />
-						{ translate( 'Secure Payment' ) }
+						{ translate( 'Secure payment' ) }
```

Comment:
:warning: This change will be queued for retranslation. We'll display the English text until then.
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **32** times:
`translate( 'Secure payment using:' )` *ES Score: 7*